Sources/SignalRClient/DefaultHttpClient.swift

```diff
@@ -63,3 +67,18 @@ class DefaultHttpClient: HttpClientProtocol {
         }
     }
 }
+
+extension DefaultHttpClient: URLSessionDelegate {
+    func urlSession(_ session: URLSession, didReceive challenge: URLAuthenticationChallenge, completionHandler: @escaping (URLSession.AuthChallengeDisposition, URLCredential?) -> Void) {
+
+        if challenge.protectionSpace.authenticationMethod == NSURLAuthenticationMethodServerTrust {
+            completionHandler(.useCredential, nil)
+        }
```

Comment:
Is this needed? If the handler is not set wouldn't `completionHandler(.performDefaultHandling, nil)` be enough? Also, because we are not returning it seems that the `completionHandler` can be called twice. Is it expected?